ui/apps/platform/src/Containers/SystemHealth/DashboardPage.js

```diff
@@ -52,9 +58,11 @@ const SystemHealthDashboardPage = () => {
                     <GridItem span={12}>
                         <BackupIntegrationHealthWidget pollingCount={pollingCountFaster} />
                     </GridItem>
-                    <GridItem span={12}>
-                        <DeclarativeConfigurationHealthCard pollingCount={pollingCountFaster} />
-                    </GridItem>
+                    {!isDeclarativeConfigHealthAvailable && (
```

Comment:
If I read that right, it should be `isDeclarativeConfigHealthAvailable &&`:
- DC health available -> show card
- DC health not available -> do not show card

Reply:
Yeah you're right, I was playing around with e2e tests and committed that on accident